kolibri/core/exams/api.py

```diff
@@ -130,3 +134,15 @@ def perform_update(self, serializer):
         if not was_archived and serializer.instance.archive:
             # It was not archived (closed), but now it is - so we set all MasteryLogs as complete
             masterylog_queryset.update(complete=True)
+
+    @action(detail=False)
+    def size(self, request, **kwargs):
+        exams = self.get_queryset()
+        exams_sizes_dict = {}
+        for exam in exams:
+            quiz_nodes = ContentNode.objects.filter(
+                id__in={source["exercise_id"] for source in exam.question_sources}
+            )
+            exams_sizes_dict[exam.id] = total_file_size(quiz_nodes)
+
+        return Response([exams_sizes_dict])
```

Comment:
Returning the map of ids to file size in a list feels a bit odd. We should either return a list of `{"id": <id>, "file_size": <file_size>}` or make it a detail endpoint that just returns the map.